CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ For experimental package changes, see the [experimental CHANGELOG](experimental/
 
 ### :bug: (Bug Fix)
 
+* fix(process-usage): added falsy check to make otel core work with browser where webpack config had process as false or null [#3613](https://github.com/open-telemetry/opentelemetry-js/issues/3613) @ravindra-dyte
```

Comment:
usually the subject (process-usage in this case) is the component being fixed, not the concept of the fix.

Reply:
Done. Replaced it with core, as suggested by @legendecas